mm2src/coins/sia.rs

```diff
@@ -373,14 +373,14 @@ impl MarketCoinOps for SiaCoin {
 
     fn min_trading_vol(&self) -> MmNumber { unimplemented!() }
 
+    fn should_burn_dex_fee(&self) -> bool { false }
```

Comment:
Please make this unimplemented for unreleased protocols.

c.c. @Alrighttt after merging this PR you should implement this part while implementing swaps. You can add it to your checklist.

Reply:
fixed